src/js/wrappers/InputWrapper.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiQuestionLine } from 'react-icons/ri';
+import { useHistory } from 'react-router-dom';
+import { Tooltip } from 'react-tippy';
+
+import Translate from 'utils/Translate';
+
+import './style.scss';
+
+const InputWrapper = ({
+  children,
+  tooltip,
+  required,
+  redirect,
+  title,
+  errorMessage,
+}) => {
+  const history = useHistory();
+
+  return (
+    <div className="input-wrapper-container">
+      <div>
+        <div className="input-wrapper-title">
+          {title && <Translate id={title?.id} defaultMessage={title?.defaultMessage} />}
+          {tooltip && (
+            <Tooltip
+              html={(
+                <span className="p-1">
+                  <Translate id={tooltip.id} defaultMessage={tooltip.defaultMessage} />
+                </span>
+              )}
+            >
+              <RiQuestionLine />
+            </Tooltip>
+          )}
+          {required && <span className="input-wrapper-asterisk">&#42;</span>}
+        </div>
+        {redirect && (
+          <div
+            onClick={() => history.push(redirect.redirectTo)}
```

Comment:
Good point, maybe a better approach (instead of passing an additional flag) will be to pass the whole "redirect" function, it will increase the usability of this button

Reply:
![image](https://github.com/openboxes/openboxes/assets/83239466/1189dc9f-cf67-4ca8-a545-5523f75add9d)
![image](https://github.com/openboxes/openboxes/assets/83239466/6aae3e6f-0502-4e05-a561-c6099dc2d42b)
![image](https://github.com/openboxes/openboxes/assets/83239466/3071bffe-2605-42ed-a299-429913797d00)
![image](https://github.com/openboxes/openboxes/assets/83239466/4c68249f-45f6-45f4-8de4-78e935082652)

Reply:
@alannadolny 

> will be to pass the whole "redirect" function, it will increase the usability of this button

I was also considering such an approach, so it'd be fine to me.